### PR TITLE
 Leaderboard: hit the API.

### PR DESCRIPTION
**Pull request by Hisoka37**
# Hi CR: 

## In this PR I : 

- [ ] Send and receive data from an API.
- [ ] Use API documentation.
- [ ] Use `arrow functions` instead of the `function` keyword.
- [ ]  Use `async` and `await` JavaScript features to consume the API.
- [ ] Follow Gitflow.

Thank you, Happy Coding 😃 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,4 +151,4 @@ This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (t-yanick) [inline README.md]: [ OPTIONAL] 
- Consider moving these lines of text as they're just a guide for you how to add the license. Now that your license is correct, this text is not really necessary again in your README

File: src/template.html [hunk 2]
```diff
@@ -14,23 +14,22 @@ <h1>Leaderboard</h1>
...
     </div>
 
     <div class="container1">
       <h2>Add your score</h2>
       <form action="">
-        <input type="text" placeholder="Your name">
-        <input type="text" placeholder="Your score">
-        <button type="submit" id="submitbtn">Submit</button>
+        <input type="text" class="name" placeholder="Your name">
+        <input type="number" id="score" placeholder="Your score">
```
Comment (t-yanick) [inline src/template.html]: [ OPTIONAL] 
- You can add proper validation to the name and score fields to ensure that a user cannot submit blank entries.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi @Hisoka37 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Great use of `arrow` functions as requested
:heavy_check_mark: `async` and `wait` with `fetch` method effectively used to send and receive data
:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file and PR

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/template.html (hunk 2 above)
Comment (Hisoka37) [reply src/template.html]: @t-yanick , thank you so much, I will add it Right now ✋🏻

**Pull request merged**